src/Symfony/Component/Mime/Address.php

```diff
@@ -31,7 +31,7 @@ final class Address
      * config, and allows to have more readable config.
      * This does not try to cover all edge cases for address.
      */
-    private const FROM_STRING_PATTERN = '~(?<displayName>[^<]*)<(?<addrSpec>.*)>[^>]*~';
+    private const FROM_STRING_PATTERN = '~(?<displayName>.*)?<(?<addrSpec>.*)>[^>]*~';
```

Comment:
what about this instead? would it allow foxing the failing test cases?
note that the comment above needs an update also
```suggestion
    private const FROM_STRING_PATTERN = '~(?<displayName>.*?)<(?<addrSpec>[^<>]*)>~';
```

Reply:
Hi everyone! 

This version does not solve the problem. This regex `~(?<displayName>.*?)<(?<addrSpec>[^<>]*)>~` crashes 3 initial tests with the following values:

- `Jane Doe <"ex<ample"@example.com>`
- `Jane Doe <"ex<amp>le"@example.com>`
- `Jane Doe > <"ex<am  p>le"@example.com>`